ga4gh/cli.py

```diff
@@ -550,34 +550,14 @@ def __init__(self, args):
         if args.readGroupIds is not None:
             self._readGroupIds = args.readGroupIds.split(",")
 
-    def _run(self, referenceGroupId, referenceId=None):
-        """
-        automatically guess reference id if not passed
-        """
-        # check if we can get reference id from rg
-        if referenceId is None:
-            referenceId = self._referenceId
-        if referenceId is None:
-            rg = self._client.getReadGroup(readGroupId=referenceGroupId)
-            iterator = self._client.searchReferences(rg.referenceSetId)
-            for reference in iterator:
-                self._run(referenceGroupId, reference.id)
-        else:
-            iterator = self._client.searchReads(
-                readGroupIds=[referenceGroupId], referenceId=referenceId,
-                start=self._start, end=self._end)
-            self._output(iterator)
-
     def run(self):
         """
         Iterate passed read group ids, or go through all available read groups
         """
-        if not self._readGroupIds:
-            for referenceGroupId in self.getAllReadGroups():
-                self._run(referenceGroupId)
-        else:
-            for referenceGroupId in self._readGroupIds:
-                self._run(referenceGroupId)
```

Comment:
The client right now "fakes" using the multiple-readGroupId functionality by requesting each readGroup individually.  Yes, we can leave it as-is if you don't want the client to take advantage of the functionality I am now implementing.